lib/Track.lua

```diff
@@ -42,7 +42,7 @@ function Track:new(x)
   t.device = 1
   -- crow
   t.pair = 1
-  t.jf = false
+  t.is_jf = false
```

Comment:
the `is_` prefix is reserved for methods. the attributes are always whatever follows the underscore.

Reply:
please change back to `t.jf`

Reply:
Changed back and modified `get_shadowable_attribute` check - see 2c04cd3